gateway/gateway.go

```diff
@@ -1102,7 +1102,7 @@ func (gateway *HandleT) Setup(application app.Interface, backendConfig backendco
 
 	gateway.batchSizeStat = gateway.stats.NewStat("gateway.batch_size", stats.CountType)
 	gateway.requestSizeStat = gateway.stats.NewStat("gateway.request_size", stats.CountType)
```

Comment:
We decided to use timer stat for request size and batch size right? Not db writes?

Reply:
done